tenable/io/v3/vm/exclusions/schema.py

```diff
@@ -0,0 +1,101 @@
+'''
+Exclusion API Endpoint Schemas
+'''
+
+from datetime import datetime
+
+from marshmallow import Schema, fields, validate
+from marshmallow.decorators import post_dump, pre_load, validates_schema
+
+
+class RulesSchema(Schema):
+    '''
+    Schema for rrules field for ScheduleSchema class
+    '''
+    freq = fields.Str(
+        validate=validate.OneOf(
+            ['ONETIME', 'DAILY', 'WEEKLY', 'MONTHLY', 'YEARLY']
```

Comment:
@alpesh-te default value for the field `freq` will be assigned while creating the schedule dictionary.
https://github.com/dhvanilshah-tenable/pyTenable/blob/exclusions/tenable/io/v3/vm/exclusions/api.py#L177

we haven't specified the default value of this field in schema because it is dependent on another boolean parameter that is `enabled` from ScheduleSchema.